from-ethereum.mdx

```diff
@@ -3,90 +3,102 @@ title: "Coming from Ethereum"
 
 Learn how to develop and build on TON coming from the Ethereum (EVM) ecosystem.
 
 ## Execution model
 
 ### Asynchronous blockchain
 
-One of the biggest stepping stones to learn TON development is the asynchronous execution model. Messages sent by one contract take time to arrive at another, meaning that the resulting transactions for processing incoming messages will occur after the current transaction terminates.
+A fundamental aspect of TON development is the asynchronous execution model. Messages sent by one contract take time to arrive at another, so the resulting transactions for processing incoming messages occur after the current transaction terminates.
 
-So, compared to Ethereum, where you can have multiple processed messages and state changes on different contracts in the same atomic transaction, a TON transaction represents a state change only for one account and single message processing. That means that a signed, included-in-block unit is called a "transaction" in both chains; however, having the differences in execution models, it has a different impact.
+Compared to Ethereum, where multiple messages and state changes on different contracts can be processed within the same atomic transaction, a TON transaction represents a state change only for one account and a single message processing. That means that a signed included-in-block unit is called a "transaction" in both chains; however, due to differences in execution models, it has a different execution effect.
 
-Here is a table for comparison:
 
 | Action description                                                                             | Ethereum                               | TON                              |
-| :--------------------------------------------------------------------------------------------- | -------------------------------------- | -------------------------------- |
+|:-----------------------------------------------------------------------------------------------|----------------------------------------|----------------------------------|
 | Single message processing with state change on one contract                                    | Message call or "internal transaction" | Transaction                      |
 | Number of state changes and messages on different accounts produced from initial contract call | Transaction                            | Chain of transactions or "trace" |
 
-Let's explore a practical example, liquidity withdrawal on DEX.
 
-On Ethereum, it will appear as a single atomic transaction with multiple contract calls inside it. You can see that this transaction has a single hash and is included in one block:
+Consider a practical example: liquidity withdrawal on a DEX.
 
-![ETH burn](/resources/images/from-ethereum/burn-tx-eth.png)
+- On Ethereum, it appears as a single atomic transaction with **multiple contract calls** inside it. This transaction has a **single hash** and is included in **one block**.
```

Comment:
Please give our style guide a thourough read: https://github.com/ton-org/docs/blob/main/contribute/style-guide-extended.mdx#emphasis-bold-and-italics

<img width="557" height="111" alt="image" src="https://github.com/user-attachments/assets/02c6dcc3-9720-4e2c-a87c-e31925b1b482" />

Reply:
It's three bold items in just two sentences.